pkg/imager/out.go

```diff
@@ -393,17 +393,10 @@ func (i *Imager) outInstaller(ctx context.Context, path string, report *reporter
 
 	printf("generating artifacts layer")
 
+	ukiPath := strings.TrimLeft(fmt.Sprintf(constants.UKIAssetPath, i.prof.Arch), "/")
+
 	if i.prof.SecureBootEnabled() {
```

Comment:
I'm confused on this condition - we should generate UKI installer for anything Talos >=1.10, shouldn't depend on SecureBoot?

And we should never put both UKI and kernel/initramfs there